pkgs/development/tools/click/default.nix

```
@@ -0,0 +1,90 @@
+{ lib
+, fetchFromGitLab
+, buildPythonPackage
+, autoconf
+, debian
+, automake
+, perl
+, libtool
+, vala
+, pkg-config
+, libgee
+, json-glib
+, properties-cpp
+, gobject-introspection
+, getopt
+, setuptools
+, pygobject3
+, wrapGAppsHook
+}:
+
+buildPythonPackage {
+  pname = "click";
+  version = "22-02-2023";
```

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

```suggestion
  version = "unstable-2023-02-22";
```